bot/helpers.py

```diff
@@ -268,9 +273,6 @@ def try_message(context, chat_id, text, reply_markup=None):
     Send a message to a user.
     """
 
-    if context.job and not context.job.enabled:
-        return
-
```

Comment:
@mr-rooftop Here `context.job.enabled` was often `False` - maybe because job was already finished before reaching this line? Why do we need this check? I removed it but please confirm that it's safe.

Reply:
As far as I understood `context.job.schedule_removal()` sets the context.job.enabled to false.
Until the scheduler actually removes the job, the job still exists with `enabled=false`.

The if check was there prevent execution of `try_message()` with a job that's set to false, because that would mean we already deleted many fields that we try to access in `try_message()`.

However, while checking out the python telegram bot library now, it seems like `context.job.schedule_removal()` doesn't set enabed to false.
Perhaps the library changed, or I'm missing something.

However, if `context.job.enabled` was often `False`, what does that mean?

I suggest trying to block the bot, and see if anything breaks.